provider/pkg/provider/chart.go

```diff
@@ -118,7 +118,7 @@ type CertManagerGlobalPodSecurityPolicy struct {
 
 type CertManagerGlobalLeaderElection struct {
 	// Override the namespace used to store the ConfigMap for leader election.
-	Namespace *string `pulumi:"namespace"`
+	Namespace pulumi.StringPtrInput `pulumi:"namespace"`
```

Comment:
I'm not familiar with how this project was set up -- do you know if this code is generated or was it written by hand?

If it's generated we probably want to create an issue somewhere (here? https://github.com/pulumi/pulumi-go-helmbase) to address the underlying bug.

Reply:
I believe its hand coded, I think @joeduffy might be the only one that knows the real answer though.